src/gui/test/screen_menu_eeprom_test.cpp

```diff
@@ -0,0 +1,216 @@
+/**
+ * @file screen_menu_eeprom_test.cpp
```

Comment:
It is redundant to put screen_menu_eeprom_test.cpp there, if the documentation is in the same file as file documented.

Reply:
I know, it is automatically generated and I use it for text searching because other methods like go to definition sometime does not work in vscode